Testscripts/Windows/WORKLOAD-HB.ps1

```diff
@@ -81,83 +122,111 @@ function Main {
 		}
 		#endregion
 
-		$fio1Command = @"
+		# setting up the workload script
+		if ($isStorageWorkloadEnable -eq 1) {
+			$work1Command = @"
 source utils.sh
 SetTestStateRunning
-fio --size=1G --name=beforehb --direct=1 --ioengine=libaio --filename=fiodata --overwrite=1 --readwrite=readwrite --bs=1M --runtime=1 --iodepth=128 --numjobs=32 --runtime=300 --output-format=json+ --output=beforehb.json
+fio --size=1G --name=beforehb --direct=1 --ioengine=libaio --filename=fiodata --overwrite=1 --readwrite=readwrite --bs=1M --runtime=1 --iodepth=128 --numjobs=32 --runtime=300 --output-format=json+ --output=beforewl.json
 rm -f fiodata
 sync
 echo 3 > /proc/sys/vm/drop_caches
 SetTestStateCompleted
 "@
-		Set-Content "$LogDir\fio1Command.sh" $fio1Command
+			Set-Content "$LogDir\work1Command.sh" $work1Command
```

Comment:
ditto, we don't need so many scripts, make the changed parts as paramter

Reply:
Fixed